### ISSUE TITLE
False positive when return statement contains assignment

### ISSUE THREAD

**Issue by mpaluchowski** [cflint/CFLint#209]
CFLint reports `MISSING_SEMI`, stating `End of statement(;) expected instead of =` for the following line:

```
return foo = bar;
```

To the best of my knowledge, an assignment in a return statement is correct code. At least Railo doesn't complain about it.

**Comment by ryaneberly** [comment]
Thanks. fixed in cfparser.
